wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
+_[Taiko no Tatsujin][Taiko no Tatsujin wikipedia]_ is a taiko drum rhythm game made by Namco. This game involves the simulation of playing the taiko drum to the beat of the music. In osu!, beatmaps can be played in the styling of Taiko no Tatsujin through the use of osu!taiko game mode.
 
-Like basic osu!, there are three elements of gameplay.
+Like osu! game mode, there are three elements of gameplay.
 
 #### Basic explanation
+##### Playfield
+![Taiko playfield][Taiko playfield image]
+
+If you have played _Taiko no Tatsujin_ before, the only difference is that there is no chibi dancers at the bottom (must be storyboarded), the pass rate is _half_ of the healthbar (not immediately obvious in the default skin; please manually find and get a decent taiko skin), Kiai Time triggers Go-Go Time, gameplay gimmick elements like hit balloons or forked paths are not implemented (only the bare-bone basics), and background display difference (osu!taiko-specific beatmaps usually has its own custom background, leaving upper section empty; a video or static background image-only will take the lower portion only; if storyboard or storyboard+video, it will take over all the available screen space except the gameplay-important parts).
+
+If you have not played _Taiko no Tatsujin_ before, the screen is split to two different section; the upper section contains the gameplay elements, and the lower section contains image/video of the beatmap. At the upper section, there is the healthbar, which unlike other gamemodes, starts off empty and must be filled it up to at least half-full to be considered a pass. At the top-middle part below the healthbar, a taiko drum is located at the left-side and a moving convenyor belt which carries the hitobjects from right to left passing through a double white circles which serves as judgement circle near the taiko drum. Above the taiko drum is the animated taiko mascot which will react based on judgement received along with upper background colour changes. Lastly, there is the usual score, accuracy, and timer of the beatmap at the upper-right.
+
+Please note that health drain is disabled in osu!taiko, so only the hitobjects will affect the healthbar. Combo does not break when you play on an empty field. Unlike osu!, the combo milestone celebration is now every 50 hits. The base score will be boosted after obtaining a combo that is a multiple of 10, but stops at 100 (this is the max combo limit for this boost). If the combo is broken, this boost resets back to the base score. During Kiai Time, every successful hit will give the player 20% more points over the current score amount. Score gained from a hit can be checked below the accuracy at the upper-right in red.
```

Comment:
Not adding the first three sentences because it is lengthy: `As mentioned earlier, your healthbar needs to be at least 50% full. This means that health drain is disabled in osu!taiko. The hitobjects are practically the only things that will affect the healthbar.`

Still using: `Please note that health drain is disabled in osu!taiko, so only the hitobjects will affect the healthbar.`

Trying to add justification for the absence of health drain?